src/test/sql/sql_pipeline_statement_test.cpp

```diff
@@ -91,6 +91,10 @@ class SQLPipelineStatementTest : public BaseTest {
   const std::string _multi_statement_query = "INSERT INTO table_a VALUES (11, 11.11); SELECT * FROM table_a";
   const std::string _multi_statement_dependant = "CREATE VIEW foo AS SELECT * FROM table_a; SELECT * FROM foo;";
 
+  const std::string _create_table = "CREATE TABLE t (a_int INTEGER)";
```

Comment:
these SQL strings "belong" to `TEST_F(SQLPipelineStatementTest, VerifyPlan) ` and the readability of that test case would increase if they were just inlined there. No need to save those strings in variables.